### PR TITLE
refactor: make the error more readble for the user (#AS-1316)

### PR DESCRIPTION
**Pull request by mlikasam-askui**

### CODE REVIEW

File: packages/askui-nodejs/src/execution/ui-control-client.ts [hunk 1]
```diff
@@ -109,12 +110,19 @@ export class UiControlClient extends FluentCommand {
...
+  }
+
   async exec(
     instruction: string,
     customElementJson: CustomElementJson[] = [],
   ): Promise<void> {
-    const customElements = await CustomElement.fromJsonListWithImagePathOrImage(customElementJson);
     const { secretText } = this;
+    const customElements = await CustomElement.fromJsonListWithImagePathOrImage(customElementJson);
+    const stringWithoutSeparators = this.escape_separator_string(instruction);
```
Comment (adi-wan-askui) [inline packages/askui-nodejs/src/execution/ui-control-client.ts:124]: You have written too much Python 😆 
```suggestion
    const stringWithoutSeparators = this.escapeSeparatorString(instruction);
```

File: packages/askui-nodejs/src/execution/ui-control-client.ts [hunk 2]
```diff
@@ -126,7 +134,7 @@ export class UiControlClient extends FluentCommand {
     } catch (error) {
       await this.annotateByDefault(TestStepState.FAILED, customElements);
       return Promise.reject(
-        new UiControlClientError(`A problem occures while executing the instruction: ${instruction}. Reason ${error}`),
+        new UiControlClientError(`A problem occures while executing the instruction: ${stringWithoutSeparators}. Reason ${error}`),
```
Comment (adi-wan-askui) [inline packages/askui-nodejs/src/execution/ui-control-client.ts:137]:
```suggestion
        new UiControlClientError(`A problem occurred while executing the instruction: ${stringWithoutSeparators}. Reason ${error}`),
```

**Review by adi-wan-askui — CHANGES_REQUESTED** [review]
Well done :)

File: packages/askui-nodejs/src/execution/ui-control-client.ts (hunk 1 above)
Comment (mlikasam-askui) [reply packages/askui-nodejs/src/execution/ui-control-client.ts:124]: das stimmit :laughing:

**Pull request merged**